### PR TITLE
Add HeatTracePipeTestEvent with code and tests

### PR DESCRIPTION
**Pull request by TordJoranger**
[AB#103047](https://statoildeveloper.visualstudio.com/b6d97460-2e53-47ae-aad4-b41396d90828/_workitems/edit/103047)

### CODE REVIEW

File: src/Equinor.ProCoSys.BusSender.Core/Models/HeatTracePipeTestEvent.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Equinor.ProCoSys.PcsServiceBus.Interfaces;
+
+namespace Equinor.ProCoSys.BusSenderWorker.Core.Models;
+#pragma warning disable CS8618
+
+public class HeatTracePipeTestEvent : IHeatTracePipeTestEventV1
+{
+    public string EventType { get; }
```
Comment (jsoi-eq) [inline src/Equinor.ProCoSys.BusSender.Core/Models/HeatTracePipeTestEvent.cs:9]: Is not in use?

**Review by terjewoldseth — APPROVED** [review]

File: src/Equinor.ProCoSys.BusSender.Core/Models/HeatTracePipeTestEvent.cs (hunk 1 above)
Comment (TordJoranger) [reply src/Equinor.ProCoSys.BusSender.Core/Models/HeatTracePipeTestEvent.cs]: Should be constant, fixed.

File: src/Equinor.ProCoSys.BusSender.Core/Services/BusEventService.cs [hunk 2]
```diff
@@ -333,6 +333,13 @@ await _eventRepository.QuerySingle<TaskEvent>(TaskQuery.GetQuery(taskId), taskId
         return JsonSerializer.Serialize(workOrderMilestoneEvent, DefaultSerializerHelper.SerializerOptions);
     }
 
+    public async Task<string?> CreateHeatTracePipeTestMessage(string message) =>
+        Guid.TryParse(message, out _)
+            ? JsonSerializer.Serialize(
+                await _eventRepository.QuerySingle<HeatTracePipeTestEvent>(
```
Comment (jsoi-eq) [inline src/Equinor.ProCoSys.BusSender.Core/Services/BusEventService.cs:339]: I find this code so compact that I have some difficulties in parsing it in my head...

File: tests/Equinor.ProCoSys.PcsBusTests/EventVersioningTests/HeatTracePipeTestEventTests.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+namespace Equinor.ProCoSys.PcsServiceBusTests.EventVersioningTests;
+
+[TestClass]
+public class HeatTracePipeTestEventTests
+{
+    [TestMethod]
+    public void IHeatTracePipeTestEventV1_InterfacePropertiesAndMethods_DoNotChange()
+    {
+        // Arrange
+        var heatTracePipeTestEventInterfaceType = typeof(IHeatTracePipeTestEventV1);
```
Comment (jsoi-eq) [inline tests/Equinor.ProCoSys.PcsBusTests/EventVersioningTests/HeatTracePipeTestEventTests.cs:16]: This is a safeguard against anyone changing the IHeatTracePipeTestEventV1 interface?

Comment (TordJoranger) [reply tests/Equinor.ProCoSys.PcsBusTests/EventVersioningTests/HeatTracePipeTestEventTests.cs:16]: Correct

**Review by jsoi-eq — COMMENTED** [review]
Waiting for feedback...

**Review by jsoi-eq — APPROVED** [review]

**Pull request merged**